Add Hawkeye by Neubird integration (ECOINT-61) (#2630)
DataDog/integrations-extras: hawkeye_by_neubird/README.md

# Hawkeye by Neubird

## Overview

> The Hawkeye App for Datadog help users accelerate their problem investigations by enabling automated investigations by Hawkeye. Connecting Datadog and Hawkeye will allow Hawkeye to look for triggered Datadog Monitors and then autonomously investigate the root cause. Once the investigation is complete the link to the investigation analysis is added to Datadog.

Reviewer: ```suggestion
The Hawkeye App for Datadog helps users accelerate their problem investigations by enabling automated investigations by Hawkeye. Connecting Datadog and Hawkeye allows Hawkeye to look for triggered Datadog Monitors and then autonomously investigate the root cause. Once the investigation is complete, the link to the investigation analysis is added to Datadog.
```

## Setup

> Here's how you can provide investigation results in Datadog with Hawkeye:

Reviewer: ```suggestion
The following steps provide investigation results in Datadog with Hawkeye:
```

# Hawkeye by NeuBird

## Overview

> Hawkeye by NeuBird accelerates incident resolution by diagnosing incidents instantly and automating root cause analysis. This integration allows Hawkeye to look for triggered Datadog Monitors and then autonomously investigate the root cause. Once investigation is complete, a link to the investigation analysis is added directly to the related Datadog incident, speeding-up resolution and reducing mean time to recovery.

Reviewer 2: Remove "instantly"

## Setup

1.  In Datadog, navigate to **Integrations**, select the Hawkeye tile and click **Install Integration**.

> 2. Click **Connect Accounts** to start the authorization process. You will be redirected to Hawkeye to complete the setup.
> 
> 3.  Once authorized, Hawkeye will begin sending investigation results to Datadog.

Reviewer: ```suggestion
2. Click **Connect Accounts** to start the authorization process. You are redirected to Hawkeye to complete the setup.

3.  Once authorized, Hawkeye begins sending investigation results to Datadog.
```

## Uninstallation

> In Datadog, navigate to **Integrations**, select the Hawkeye tile and click **Uninstall Integration**.

Reviewer: ```suggestion
In Datadog, navigate to **Integrations**, select the Hawkeye tile, and click **Uninstall Integration**.
```
